subjects/java/piscine/AdventureSorcerer/README.md

```diff
@@ -13,10 +13,10 @@ It has a constructor with three parameters :
 
 From the `Healer` interface : 
 * `getHealCapacity` returns the property `healCapacity`
-* `heal` add `healCapacity` to the currentHealth of the `Character` in parameter. Beware that the `currentHealth` can't be greater the `maxHealth`. You may need to change the accessibility of the `currentHealth` param in the `Character` to be able to update the value.
+* `heal` adds `healCapacity` to the currentHealth of the `Character` in parameter. Beware that `currentHealth` can't be greater the `maxHealth`. You may need to change the accessibility of the `currentHealth` parameter in  `Character` class to be able to update the value.
```

Comment:
attribute instead of parameter here `currentHealth` parameter should be `currentHealth` attribute

Reply:
make sure to remove the additional space from this line, try to use chatGpt for that